scripts/auto_import/auto_import.rb

```diff
@@ -0,0 +1,24 @@
+#!ruby -w
+require 'erb'
+
+raise 'must provide a library to import' if ARGV[0].nil?
+LIB = ARGV[0]
+RUBY = `which ruby`.strip
+source_file = `gem which #{LIB}`.strip # used in erb
```

Comment:
Right, but my goal is to let ruby resolve where the gem is located. Are you propose giving a path to gem?

Reply:
Does something like this work?

```ruby
package = 'uri'
require package
base = RbConfig::CONFIG['rubylibprefix']
lib_sources = $LOADED_FEATURES.select { |f| f.start_with?(base) }
package_sources = lib_sources.select { |f| f =~ %r{/#{package}} }
puts package_sources.sort
```

Reply:
Using this approach and enumerating all of the source files will allow us to avoid depending on rust-embed in artichoke-backend and also makes sure we aren't pulling in any cruft.

Reply:
This seems easy to address in a followup once we want to use this to support the multi-file libs. I don't consider addressing this feedback blocking.